spec/models/school_spec.rb

```diff
@@ -4,6 +4,27 @@
   it { expect(subject.attributes).to include('gias_data') }
   it { expect(described_class.columns_hash['gias_data'].type).to eql(:json) }
 
+  describe '#has_religious_character?' do
+    it 'returns false when the school has no gias_data' do
+      subject.gias_data = nil
+      expect(subject.has_religious_character?).to be false
+    end
+
+    before do
```

Comment:
Move this at the top of the describe block - it's not positional, so it will apply to everything in the block.